JS

```diff
@@ -0,0 +1,211 @@
+
+function Hamburger(type, count,stuffing){
+  this.name = this[type].name;
+  this.count = count;
+  this.stuffingName = this[stuffing].name;
+  this.price = this[type].price + this[stuffing].price;
+  this.calories = this[type].calories + this[stuffing].calories;
+}
+
+Hamburger.prototype.LARGE_SIZE= { 
```

Comment:
@Mikleprog It's much more better and common way to store constants like properties of class, f.e.:
Hamburger.LARGE_SIZE = { ... }
and in prototypes store only common methods